### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by IgorBochkor**
added solution

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/MovieSessionDao.java [hunk 1]
```diff
@@ -12,4 +13,6 @@
     MovieSession update(MovieSession session);
 
     void deleteById(Long id);
+
+    Optional<MovieSession> get(Long id);
```
Comment (Baldinini) [inline src/main/java/com/dev/cinema/dao/MovieSessionDao.java]:
```suggestion
    Optional<MovieSession> getById(Long id);
```

File: src/main/java/com/dev/cinema/dao/UserDao.java [hunk 2]
```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    Optional<User> get(Long id);
```
Comment (Baldinini) [inline src/main/java/com/dev/cinema/dao/UserDao.java]: same

File: src/main/java/com/dev/cinema/service/MovieSessionService.java [hunk 3]
```diff
@@ -12,4 +12,6 @@
     MovieSession update(MovieSession session);
 
     void deleteById(Long id);
+
+    MovieSession get(Long id);
```
Comment (Baldinini) [inline src/main/java/com/dev/cinema/service/MovieSessionService.java]:
```suggestion
    MovieSession getById(Long id);
```

File: src/main/java/com/dev/cinema/service/UserService.java [hunk 4]
```diff
@@ -7,4 +7,6 @@
     User add(User user);
 
     Optional<User> findByEmail(String email);
+
+    User get(Long id);
```
Comment (Baldinini) [inline src/main/java/com/dev/cinema/service/UserService.java]: same

**Review by Baldinini — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/service/UserService.java (hunk 4 above)
Comment (IgorBochkor) [reply src/main/java/com/dev/cinema/service/UserService.java]: fixed

File: src/main/java/com/dev/cinema/service/MovieSessionService.java (hunk 3 above)
Comment (IgorBochkor) [reply src/main/java/com/dev/cinema/service/MovieSessionService.java]: fixed

File: src/main/java/com/dev/cinema/dao/UserDao.java (hunk 2 above)
Comment (IgorBochkor) [reply src/main/java/com/dev/cinema/dao/UserDao.java]: fixed

File: src/main/java/com/dev/cinema/dao/MovieSessionDao.java (hunk 1 above)
Comment (IgorBochkor) [reply src/main/java/com/dev/cinema/dao/MovieSessionDao.java]: fixed

**Review by Baldinini — APPROVED** [review]

File: src/main/java/com/dev/cinema/dao/MovieSessionDao.java (hunk 1 above)
Comment (boroda4436) [reply src/main/java/com/dev/cinema/dao/MovieSessionDao.java]: @Baldinini It's ok to have a `get(Long id)` method name in DAO. In this case, you understand what ID is (the unique identifier)

File: src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
+package com.dev.cinema.model.dto;
+
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long orderId;
+    private List<Long> tickets;
+    private Long userId;
+    private String email;
```
Comment (boroda4436) [inline src/main/java/com/dev/cinema/model/dto/OrderResponseDto.java]:
```suggestion
    private String userEmail;
```

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**